html/janus.js

```diff
@@ -649,7 +638,7 @@ function Janus(gatewayCallbacks) {
 			if(config.pc && config.remoteSdp) {
 				// Add candidate right now
 				Janus.debug("Adding remote candidate:", candidate);
-				if(!candidate || candidate.completed === true) {
+				if(!candidate || candidate.completed) {
```

Comment:
In the Janus API, `completed` in `trickle` is a boolean, so I think the `true` check we had was correct and strict as it was supposed to be.